data-r2dbc/src/test/groovy/io/micronaut/data/r2dbc/postgres/PostgresRepositorySpec.groovy

```diff
@@ -158,4 +158,16 @@ class PostgresRepositorySpec extends AbstractRepositorySpec implements PostgresT
         return true
     }
 
+    void "test @Where annotation"() {
```

Comment:
this test failed with an exception before the change in AbstractSqlLikeQueryBuilder.java